Changelog.md

```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented here.
 - Update python, pyta and jupyter testers to allow a requirements file (#580)
 - Update R tester to allow a renv.lock file (#581)
 - Improve display of Python package installation errors when creating environment (#585)
+- Update "setting up test environment" message with http response (#589)
```

Comment:
It is good to be precise about which http status code is used (this is the whole point of this change). Also please note the difference between "response" and "status code".